explorer/src/lib/components/navbar/Navbar.svelte

```diff
@@ -0,0 +1,129 @@
+<script>
+  import { NavList, Select } from "$lib/dusk/components";
+  import "./styles.css";
+
+  /** @type {Number}*/
+  let clientWidth;
+
+  /** @type {HTMLBodyElement}*/
+  let body;
+
+  let hidden = true;
+
+  const navigation = [
+    {
+      link: "/",
+      title: "Chain Info",
+    },
+    {
+      link: "/blocks",
+      title: "Blocks",
+    },
+    {
+      link: "/transactions",
+      title: "Transactions",
+    },
+  ];
+
+  const networks = [
+    {
+      label: "testnet",
+      value: `${import.meta.env.VITE_DUSK_TESTNET_NODE}`,
+    },
+    {
+      label: "devnet",
+      value: `${import.meta.env.VITE_DUSK_DEVNET_NODE}`,
+    },
+  ];
+
+  //@ts-ignore;
+  const bindBody = (node) => {
+    body = node;
+  };
+
+  /**
+   * @param {Boolean} bool
+   */
+  const toggleScroll = (bool) => {
+    if (!body) {
+      return;
+    }
+
+    if (bool) {
+      body.style.overflow = "auto";
+    } else {
+      body.style.overflow = "hidden";
+    }
+  };
+
+  $: toggleScroll(hidden);
+</script>
+
+<svelte:body use:bindBody />
+<svelte:window bind:innerWidth={clientWidth} />
+
+<nav class="dusk-navbar" class:dusk-navbar--menu-hidden={hidden}>
+  <a href="/" class="dusk-navbar__logo">
```

Comment:
I'm not using the Anchor here as I'd have to overwrite all the css that the Anchor has in order for it to satisfy the needs imposed by the design.

Reply:
> Also in the README you mention a `VITE_BASE_PATH` env variable: if the explorer might be served from a sub-folder like the wallet, then you should import the other utilities we wrote for the wallet, to avoid making too many changes down the line with the risk of introducing bugs.

You have to refresh my memory here, what other utilities?